internal/services/controllers/events/controller.go

```diff
@@ -194,6 +198,13 @@ func (ec *EventsControllerImpl) processEvent(ctx context.Context, tenantId, even
 
 		g.Go(func() error {
 
+			if additionalMetadata["hatchet__event_id"] == nil {
```

Comment:
Do we restrict the user from setting/viewing this? If not, we definitely should.

Reply:
@abelanger5 @grutt I added this here as an initial pass at the stream by additional metadata thing we talked about gabe. It sounds like we aren't going down that route? Should I just remove it ? Also I'm not clear on the problems with doing this, is it that we might be allowing users to change parent ids?

Reply:
i could see a potential exfiltration issue if users can set hatchet__event_id and then we later fetch a different tenant's data for a stream event for example

i dont see an issue with letting users read this though unless im missing something... i think this would actually be kind of convenient

maybe we just set any keys prefixed with `hatchet__` to nil first?